website_sale_product_brand/README.rst

```diff
@@ -33,21 +37,35 @@ Based on this configuration, you will see the menuitem shop by brand under shop
 It will show all the brands and once you select that brand it will show product's which
 is related to this brand.
 
+The blog here explains the HOWTO :
```

Comment:
I remember a comment about it: should not we rather have the content of the blog in here? (Or at least a summary)